src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -1047,15 +1045,6 @@ public static void addSignalHandler(final SignalHandler signalHandler) {
     }
 
 
-    public interface SignalHandler {
```

Comment:
possibly a breaking API change!

Reply:
This interfaces was moved to after the static methods